tests/testcases/040_check-network-adv.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: all
+  tasks:
+
+    - name: Test tunl0 routes
+      shell: "! ip ro | grep '/26 via' | grep -v tunl0"
+      when: ipip|default(false)
+
+- hosts: kube-master
+  tasks:
+
+    - name: Get services
+      shell: kubectl get svc --namespace=default
+      register: get_svc
+      run_once: true
+
+    - name: Deploy netchecker
+      shell: "curl -s https://raw.githubusercontent.com/bogdando/netchecker-server/dev/utils/deploy_netchecker.sh | bash"
```

Comment:
I would really prefer to keep this file located somewhere in kargo land so we can debug it. curl + bash isn't the best route to go. Can you move the file in here before merging?